### DISCUSSION TITLE
Flexibility of File/Image Tables

### DISCUSSION THREAD

**Discussion by cr101** [epicweb-dev/epic-stack#165] [Q&A]
Hi,

i’m planning to build a web app which will have an `Organizations` model.

Is it possible to store the logo images for the `Organizations` in the existing File/Image database tables?

**Comment by kentcdodds** [comment]
Yep 👍 the relationship will be similar to the one for users.

**Reply by cr101** [reply]
Cool. Hopefully Prisma will eventually add [Polymorphic relationships](https://github.com/prisma/prisma/issues/1644) OR [union types](https://github.com/prisma/prisma/issues/2505)
